common/test/acceptance/pages/lms/instructor_dashboard.py

```diff
@@ -839,6 +839,7 @@ def submit_allowance_form(self, allowed_minutes, username):
         self.q(css='input#user_info').fill(username)
         self.q(css="input#addNewAllowance").click()
         self.wait_for_element_absence("div.modal div.modal-header", "Popup should be hidden")
+        self.wait_for_ajax()
```

Comment:
@benpatterson It is completely possible for element initiating ajax disappear before ajax call completes. It is due to the fact that Ajax call are asynchronous by nature. Ajax calls have callback methods which are called based on success or failure of Ajax calls. In our case; given the nature of Ajax calls (that is async calls),  Ajax calls may continue even click button has disappeared, thats why I put it after the wait_for_element_absence. Just to let you know, I have tested it both (by putting wait_for_ajax before and after wait_for_element_absence) ways and test doesn't fail. It is your call whether we should place it before or after wait_for_element_absence